### PR TITLE
swap two file targets to be object targets

### PR DESCRIPTION
**Pull request by Kfaunce**
image

### CODE REVIEW

File: 1_fetch/src/get_site_info_data.R
Comment (lindsayplatt) [inline 1_fetch/src/get_site_info_data.R:6]: I don't see anything in this function called `site_list`. I think you still want to keep line 7 so that you can get a vector of `sites` from the data to use here. I think you could just update to `sites <- unique(eval_data$site_no)` since you no longer need to read in a CSV file.

**Review by lindsayplatt — COMMENTED** [review]
Looks good! One small comment about the `get_site_info_data()` function that I am not sure is working as you may expect. I think one of your files failed, so you may need to try running `tar_make()` a few times to make sure they all run. It should skip the ones that succeeded and just build the ones that failed.

File: 1_fetch/src/get_site_info_data.R
Comment (Kfaunce) [reply 1_fetch/src/get_site_info_data.R]: I created the `site_list` variable in `_targets.R`, but I updated this function to look at the unique site numbers in the combined file instead, as you suggested.

Comment (lindsayplatt) [reply 1_fetch/src/get_site_info_data.R]: Right, if `site_list` is in `_targets.R`, you should pass it _into_ this function as an argument. So, I would suggest declaring this function as `get_site_info_data <- function(eval_data, site_list)` and then changing your targets code that calls the function to pass in this second argument.

Comment (Kfaunce) [reply 1_fetch/src/get_site_info_data.R:6]: Oh! Ok, this makes sense. Went ahead and did this and pushed those updates.

**Review by lindsayplatt — APPROVED** [review]
I'll let you hit the merge button!

**Pull request merged**
